CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
-v2.0
+# Changelog
 
-- Major Project Split. Please see the main ExAws repository for previous changelogs.
+## Unreleased
```

Comment:
@kianmeng  Are you sure these are unreleased? The lastest release is v4.0.0, which is commit e787070 - I only see one commit after that, in which I've added an on-push workflow. I may be wrong here - if you could explain and help me understand, I'd be happy to help verify one way or the other.

Reply:
@darrenklein You're right, I've updated the Changelog accordingly.